docs/spec/v1beta2/ocirepositories.md

```diff
@@ -555,6 +557,14 @@ For publicly available OCI artifacts, which are signed using the
 [Cosign Keyless](https://github.com/sigstore/cosign/blob/main/KEYLESS.md) procedure,
 you can enable the verification by omitting the `.verify.secretRef` field.
 
+To verify that the subject and the OIDC issuer present in the Fulcio certificate
+you can specify `.spec.verify.matchOIDCIdentity`. It provides two fields:
+
+- `.issuer`, to sepcify a regexp that matches against the OIDC issuer.
```

Comment:
```suggestion
- `.issuer`, to specify a regexp that matches against the OIDC issuer.
```
Typo exists in other places too, try to use a spellchecker in your IDE. 
Please try to use